### PR TITLE
Relax the rule that tries to get rid of this./me. in Roslyn code.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/36874

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @mavasani @jasonmalinowski 

There are roughly 1k instances of this just in the IDE alone.

File: .editorconfig [hunk 1]
```diff
@@ -44,10 +44,10 @@ dotnet_diagnostic.xUnit2018.severity = suppress # "do not compare an object's ex
...
 dotnet_sort_system_directives_first = true
 # Avoid "this." and "Me." if not necessary
-dotnet_style_qualification_for_field = false:suggestion
-dotnet_style_qualification_for_property = false:suggestion
-dotnet_style_qualification_for_method = false:suggestion
-dotnet_style_qualification_for_event = false:suggestion
+dotnet_style_qualification_for_field = false:none
+dotnet_style_qualification_for_property = false:none
+dotnet_style_qualification_for_method = false:none
+dotnet_style_qualification_for_event = false:none
```
Comment (sharwell) [inline .editorconfig]: ❗️  These should be `silent` (or `refactoring`) so the fading and Fix All still works.

Comment (CyrusNajmabadi) [reply .editorconfig]: Done.

**Comment by CyrusNajmabadi** [comment]
@mavasani this all passed.  can you merge?

**Comment by CyrusNajmabadi** [comment]
Thanks!

**Comment by devin-fee-ah** [comment]
Fingers crossed someone responds to this, but this appears to be incorrect. Alternatively, the severity level of `refactoring` is undocumented: https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-language-conventions?view=vs-2019#severity-levels

Any thoughts?

**Comment by sharwell** [comment]
@devin-fee-ah It was introduced in #28043 as a synonym for `silent`

**Pull request merged**